runtime/near-vm-runner/src/imports.rs

```diff
@@ -103,7 +103,7 @@ imports! {
     sha256<[value_len: u64, value_ptr: u64, register_id: u64] -> []>,
     keccak256<[value_len: u64, value_ptr: u64, register_id: u64] -> []>,
     keccak512<[value_len: u64, value_ptr: u64, register_id: u64] -> []>,
-    ed25519_verify<[ sig_len: u64,
+    ##["protocol_feature_ed25519_verify"] ed25519_verify<[ sig_len: u64,
```

Comment:
@matklad is this the right way of doing it?

Reply:
Almost! It should read

```
#["protocol_feature_ed25519_verify", Ed25519Verify]
```


The first component is compile-time switch, so that the code isn't included in the binary at all while the feature is not stable. 

The second component is the runtime switch, so that the function is only included in the recent blocks, and doesn't become retroactively available to old blocks.

Reply:
Thanks!